neo/UInt160.cs

```diff
@@ -31,26 +31,41 @@ public UInt160(byte[] value)
         /// Method CompareTo returns 1 if this UInt160 is bigger than other UInt160; -1 if it's smaller; 0 if it's equals
         /// Example: assume this is 01ff00ff00ff00ff00ff00ff00ff00ff00ff00a4, this.CompareTo(02ff00ff00ff00ff00ff00ff00ff00ff00ff00a3) returns 1
         /// </summary>
-        public int CompareTo(UInt160 other)
+        public unsafe int CompareTo(UInt160 other)
         {
-            byte[] x = ToArray();
-            byte[] y = other.ToArray();
-            for (int i = x.Length - 1; i >= 0; i--)
+            fixed (byte* px = ToArray(), py = other.ToArray())
             {
-                if (x[i] > y[i])
-                    return 1;
-                if (x[i] < y[i])
-                    return -1;
+                uint* lpx = (uint*)px;
+                uint* lpy = (uint*)py;
+                //160 bit / 32 bit step   -1
+                for (int i = (160 / 32 - 1); i >= 0; i--)
```

Comment:
I thought we were going to unroll to use 1 `uint` comparison and 2 `ulong` comparisons instead of 5 `unit` comparisons?

Reply:
yes,can do more test to find a fast way.

Reply:
Can we resolve this, @jsolman? Maybe we could add this change in another PR after another tests.

Reply:
It can be switched later

Reply:
I dismissed the review; this comment can be left open though while the PR can still be approved.